src/elements/menu/menu/menu.scss

```diff
@@ -23,9 +23,6 @@
   --sbb-menu-min-width: 100%;
   --sbb-menu-inset: 0 auto auto 0;
   --sbb-menu-container-height: 100dvh;
-  --sbb-menu-open-animation-name: open;
-  --sbb-menu-close-animation-name: close;
-  --sbb-menu-back-button-display: block;
```

Comment:
Out of curiosity, why did you change this?

Reply:
Davide suggested it, and I think it makes sense